src/core/lib/experiments/experiments.yaml

```diff
@@ -55,14 +55,14 @@
 - name: canary_client_privacy
```

Comment:
Just to check... when can we remove this one (and it's friends?)

Reply:
I saw a change from the DDT team recently that still relied on the experiments, I think probably end of year - I'll check with them.